src/Options.tsx

```diff
@@ -128,7 +135,7 @@ const Options: React.FC<OptionsProps> = (props) => {
       <Select
         disabled={disabled}
         prefixCls={selectPrefixCls}
-        showSearch={false}
+        showSearch={showSearch}
         className={`${prefixCls}-size-changer`}
         optionLabelProp="children"
         popupMatchSelectWidth={false}
```

Comment:
Could we pass the whole showSizeChanger object to Select? That we could customize Select as needed.

```jsx
showSizeChanger={{
  disabled: true,
}}
```